docs/faq/faq.md

```diff
@@ -255,3 +255,12 @@ Our `ink_storage` data structures on the other hand are optimized for storage an
 differentiation between lazy and eager access.
 
 [See this chapter](/datastructures/overview) where we go into more details and provide examples.
+
+
+### Why am I getting a `ContractTrapped` error when interacting with a contract?
+
+`ContractTapped` is returned in case the contract reached an unreachable instruction. 
+This could e.g. be caused by an `unreachable!` instruction, a failed assertion or a `panic!` instruction.
+
+It either constitutes a bug in your contract or in ink!. In case you think
+this is a bug in ink!, [please create an issue](https://github.com/paritytech/ink/issues/new/choose).
```

Comment:
```suggestion
When it does not constitute a deliberate assertion like for example a permission check it is most likely a
bug in your contract or in ink!. In case you think this is a bug in ink!, [please create an issue (https://github.com/paritytech/ink/issues/new/choose).
```

Reply:
We maybe should also mention that integer overflows can cause contract trapped and provide hints to disabled them in `Cargo.toml` with `overflow-checks = false`.

Reply:
I kinda fear that many people will instantly open an issue in ink! repo instead of trying to figure out if the problem is caused by them as a reflex reaction when they see this.